pex/bin/pex.py

```diff
@@ -444,6 +445,17 @@ def configure_clp_pex_environment(parser):
            'string representing the python version (e.g. "27", "36"). ABI is the ABI tag '
            '(e.g. "cp36m", "cp27mu", "abi3", "none"). Default: current platform.')
 
+  group.add_option(
+      '--resolve-local-platforms',
+      dest='resolve_local_platforms',
+      default=False,
```

Comment:
Is the default `False` for backward-compatibility or because you think it's the more sound default?

Reply:
Both with the added reason that its slow in a cold cache case. On my machine with ~10 interpreters to search, building the cold interpreter cache takes ~1 second. After that these checks are ~20ms though.